ex5.sh

```diff
@@ -0,0 +1,34 @@
+# q1:
+touch File{A..F}.txt
+
+# q2:
+rm File{C..F}.txt
+
+# q3:
+mv FileA.sh FileA.old
+
+# q4:
+rm *
+
+# q5:
+cp /etc/*.conf subs 2>/dev/null
+
+# q6:
+cat l*
+
+# q7:
+grep -l "user" ????[^.][^.][^.].conf
+
```

Comment:
Removed the [^.][^.][^.] pattern from the command in q7 as it was not necessary. The ???? pattern alone would match four-letter filenames without considering the extension.